fedbiomed/researcher/aggregators/aggregator.py

```diff
@@ -154,7 +154,7 @@ def save_state_breakpoint(
 
     def _save_arg_to_file(self, breakpoint_path: str, arg_name: str, node_id: str, arg: Any) -> str:
 
-        filename = os.path.join(breakpoint_path, f"{arg_name}_{node_id}.mpk")
+        filename = os.path.join(breakpoint_path, f"{arg_name}_{str(node_id)[0:11]}_{uuid.uuid4()}.mpk")
```

Comment:
Why do you select only the first 11 characters of NODE ID?

Reply:
Thank you @sharkovsky for the review !

Thi is to keep the user readable information of the node's name, but with a short enough file name.
This also spots that the node name cannot be considered as a unique value for these files